src/DBImport.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+namespace TheRealMVP;
+
+class DBImport
+{
+    protected $pdo;
+
+    /**
+     * DBImport constructor.
+     *
+     * Creates new PDO object
+     */
+    public function __construct()
+    {
+        $this->pdo = new \PDO ("mysql:host=db; dbname=TheRealMVP", "root", "password");
+    }
+
+    /**
+     * Drops all tables and recreates them
+     *
+     * Populates all tables
+     *
+     * @param $apiData
+     *                Array of data from api
+     */
+    public function storeData(array $apiData): void
```

Comment:
Would you ever want to use the API class outside of importing the properties?

I'd suggest instead of calling the API class in index.php, and then taking the result and passing it into this method, that you instead actually store an instance of a GetAPI object as a property on this class (passing it in via the constructor), you could then directly call the getJson method directly in here.